frame/system/benchmarking/res/README.md

```diff
@@ -0,0 +1,3 @@
+These runtimes are used for benchmarking the `set_code` intrinsic.
+
+**Don't use them in production environments!**
```

Comment:
Could you please add instructions on how to update this runtime file?  
I imagine we want to update it every few months or so.